### PR TITLE
Expand Azure pipeline

### PR DESCRIPTION
**Pull request by sethaxen**

### CODE REVIEW

File: azurepipelines-install_cmdstan.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  type: string
+  default: ''
+
+steps:
+- bash: |
+    set -o xtrace
+    wget -P ${{ parameters.dir }} https://github.com/stan-dev/cmdstan/releases/download/v${{ parameters.version }}/cmdstan-${{ parameters.version }}.tar.gz
+    tar -xzpf ${{ parameters.dir }}/cmdstan-${{ parameters.version }}.tar.gz -C ${{ parameters.dir }}
+    make -C ${{ parameters.dir }}/cmdstan-${{ parameters.version }}/ build
+    displayName: 'Download and build CmdStan'
```
Comment (ahartikainen) [inline azurepipelines-install_cmdstan.yml]: Indent back

File: azurepipelines-install_julia.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    then
+      wget -nv https://julialangnightlies-s3.julialang.org/bin/linux/x64/julia-latest-linux64.tar.gz
+      mkdir julia-${{ parameters.version }}
+      tar zxf julia-latest-linux64.tar.gz -C julia-${{ parameters.version }} --strip-components 1
+    else
+      wget -nv https://julialang-s3.julialang.org/bin/linux/x64/${{ parameters.version }}/julia-${{ parameters.version }}-latest-linux-x86_64.tar.gz
+      mkdir julia-${{ parameters.version }}
+      tar zxf julia-${{ parameters.version }}-latest-linux-x86_64.tar.gz -C julia-${{ parameters.version }} --strip-components 1
+    fi
+    displayName: 'Download and extract Julia'
```
Comment (ahartikainen) [inline azurepipelines-install_julia.yml]: Indent back

File: azure-pipelines.yml [hunk 3]
```diff
@@ -5,15 +5,72 @@
 
 trigger:
 - master
+- releases/*
+
+variables:
+  PYTHON: "Conda" # use Julia's packaged Conda build for installing packages
+  CMDSTAN_VERSION: "2.21.0"
```
Comment (ahartikainen) [inline azure-pipelines.yml]: 2.22.0 is the latest version.

CmdStanPy has the following script to install the latest version

https://github.com/stan-dev/cmdstanpy/blob/master/cmdstanpy/install_cmdstan.py

Comment (sethaxen) [reply azure-pipelines.yml]: At the moment, CmdStan.jl still tests 2.21.0 with CI. Once it begins testing against 2.22.0, I'll happily update.

**Pull request merged**